tests/delete/tests.py

```diff
@@ -102,6 +102,23 @@ def test_do_nothing_qscount(self):
             b.delete()
         self.assertEqual(Base.objects.count(), 0)
 
+    @skipUnlessDBFeature("supports_foreign_keys")
+    def test_db_cascade(self):
+        a = create_a('db_cascade')
+        a.db_cascade.delete()
+        self.assertFalse(A.objects.filter(name='db_cascade').exists())
```

Comment:
I think we shouldn't reuse the `A` model for testing `DB_*` delete options, it also raises `delete.A.db_cascade: (fields.W346)`.